adserver/urls.py

```diff
@@ -143,6 +144,14 @@
         name="publisher_uplift_report_export",
     ),
     # Advertiser management and reporting
+    path(
+        r"advertiser/advertisement-bulk-create-template.csv",
+        TemplateView.as_view(
+            template_name="adserver/advertiser/advertisement-bulk-create-template.csv",
+            content_type="text/csv",
+        ),
+        name="advertiser_bulk_create_template",
+    ),
```

Comment:
Could also just serve this via static files? I don't feel too strongly, but this isn't a pattern I think we're using elsewhere.

Reply:
It isn't but it keeps it self contained rather than a file that might get deleted. Or were you talking about local static files? I guess I could do that and just add another directory to `STATICFILES_DIRS`

https://github.com/readthedocs/ethical-ad-server/blob/c84adf39875cb18a829387b6b0dd72bcbf4bce87/config/settings/base.py#L272C1-L275

Reply:
You could just put it in the existing static files directory, right?

Reply:
The only static files directories we have are for images and build JS. I'll just create a new one.